fem/bilininteg_mass_pa.cpp

```diff
@@ -440,8 +440,33 @@ static void PAMassAssembleDiagonal(const int dim, const int D1D,
 
 void MassIntegrator::AssembleDiagonalPA(Vector &diag)
 {
-   if (pa_data.Size()==0) { SetupPA(*fespace, true); }
-   PAMassAssembleDiagonal(dim, dofs1D, quad1D, ne, maps->B, pa_data, diag);
+#ifdef MFEM_USE_CEED
```

Comment:
These lines are exactly the same as in `DiffusionIntegrator`. Is there a way to avoid the copy-paste and reuse this code?

Reply:
How about this approach: 1066cda2956151d45f5903e81bf055d902b5c462

Reply:
That's much nicer!